package.json

```diff
@@ -187,6 +190,10 @@
                         "disable-linters": {
                             "type": "array",
                             "description": "List of linters by name to disable"
+                        },
+                        "resource-context-menu": {
```

Comment:
This name feels a bit opaque.  `resource-commands-on-files` perhaps?

Reply:
done.